apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -55,8 +55,33 @@ function renderSectionProgress(section, validScripts) {
   registerReducers({sectionProgress});
   const store = getStore();
   store.dispatch(setSection(section));
-  store.dispatch(setValidScripts(validScripts));
 
+  if (experiments.isEnabled('courseVersions')) {
```

Comment:
is this a new experiment? we've found it helpful to define new constants in `experiments.js` and then import them.